task/10-katas-1-tasks.js

```diff
@@ -56,7 +79,44 @@ function createCompassPoints() {
  *   'nothing to do' => 'nothing to do'
  */
 function* expandBraces(str) {
-    throw new Error('Not implemented');
+    function parse(str) {
+
+
+        let open = str.lastIndexOf('{'),
+            close = str.slice(open, str.length).indexOf('}') + open;
+
+
+
+        if (open == -1 || close == -1) {
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"